client/components/domains/layout/two-columns-layout.tsx

```diff
@@ -0,0 +1,19 @@
+import classNames from 'classnames';
+import React from 'react';
+import './style.scss';
+import { TwoColumnsLayoutProps } from './types';
+
+const TwoColumnsLayout = ( {
+	className = '',
+	content,
+	sidebar,
+}: TwoColumnsLayoutProps ): JSX.Element => {
+	return (
+		<div className={ classNames( 'two-columns-layout', className ) }>
```

Comment:
Is this layout different from what we would get using a [`Layout` with two `Column`s](http://calypso.localhost:3000/devdocs/design/layout)? 
[Here's](https://github.com/Automattic/wp-calypso/pull/58662) an implementation example:

https://github.com/Automattic/wp-calypso/blob/c55c7c27b448551c173339122d430613670e2f86/client/my-sites/domains/domain-management/transfer/transfer-page/index.tsx#L69-L116

I think these could be reused in here, but if that's not the case, then feel free to ignore this comment 😄

Reply:
Yeah right, I didn't know that component but I've seen that the spaces in Figma are just a little bit different. My idea is to uniform two columns layout for all domains pages (maybe even subcomponents of sidebar, such as message box). I discussed this with Del and created a task in Asana 👍. Thanks

Reply:
While the style is a little different, I thought it was a good idea to reuse it in your `TwoColumnsLayout` component - keeping consistency between our pages and reusing the already implemented component. 
But I suppose that's minor - and even if that's the case, we can still do it in separate PRs.

Reply:
I totally agree 😊